crates/sui-e2e-tests/tests/checkpoint_tests.rs

```diff
@@ -44,6 +44,14 @@ async fn basic_checkpoints_integration_test() {
 
 #[sim_test]
 async fn checkpoint_split_brain_test() {
+    #[cfg(msim)]
+    {
+        use sui_core::authority::{init_checkpoint_timeout_config, CheckpointTimeoutConfig};
+        init_checkpoint_timeout_config(CheckpointTimeoutConfig {
+            timeout: Duration::from_secs(2),
+            panic_on_timeout: false,
```

Comment:
Can we comment on if not making progress is expected, in case someone wants to investigate in future?